src/main/java/com/karankumar/bookproject/ui/settings/SettingsView.java

```diff
@@ -131,9 +139,9 @@ private String generateJsonResource() {
             final StreamResource resource = new StreamResource("bookExport.json",
                     () -> new ByteArrayInputStream(jsonRepresentationForBooks));
             final StreamRegistration registration = UI.getCurrent()
-                                                    .getSession()
-                                                    .getResourceRegistry()
-                                                    .registerResource(resource);
+                    .getSession()
+                    .getResourceRegistry()
+                    .registerResource(resource);
```

Comment:
This will fixes the indentation (suggested by intellj). Should I undo this change?
please let me know

Reply:
Yes, we have our [own custom IntelliJ code style file](https://github.com/Project-Books/book-project/blob/master/ide/intellij/book_project_code_style.xml) that you can import (it may not currently work due to a bug. There's an open issue for this)

Reply:
Done